ramls/automated-patron-blocks.raml

```diff
@@ -11,27 +11,80 @@ types:
   user: !include user.json
   loanPolicy: !include loan-policy.json
   loan: !include loan.json
+  account: !include accountdata.json
   errors: !include raml-util/schemas/errors.schema
   automatedPatronBlocks: !include automated-patron-blocks.json
+  synchronizationRequest: !include synchronization-request.json
+  synchronizationResponse: !include synchronization-response.json
```

Comment:
This should be the same object - SynchronizationJob

Reply:
done